build/emoji/emoji_gen.mjs

```diff
@@ -40,6 +40,7 @@ readDirPromise.then((images) => {
         ).catch((err) => console.log(`[ERROR] Failed to copy ${imageFile}: ${err}`)));
     }
 });
+Fs.copyFile('images/icon64x64.png', 'images/emoji/mattermost.png');
```

Comment:
love this, we don't have to worry the emoji being not the same if we improve over it